src/wazuh_modules/wm_gcp.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Wazuh Module for Security Configuration Assessment
+ * Copyright (C) 2015-2019, Wazuh Inc.
+ * November 25, 2018.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef WM_GCP_H
+#define WM_GCP_H
+
+#define WM_GCP_LOGTAG ARGV0 ":gcp-pubsub"
+#define WM_GCP_DEFAULT_DIR WM_DEFAULT_DIR "/gcp-pubsub"
```

Comment:
Change `WM_GCP_DEFAULT_DIR` and `WM_GCP_SCRIPT_PATH` according to changes at https://github.com/wazuh/wazuh/pull/4261